test/src/androidTest/kotlin/io/realm/InstrumentedTests.kt

```diff
@@ -127,4 +141,95 @@ class InstrumentedTests {
             }
         }
     }
+
+    @org.junit.Test
```

Comment:
I think these tests a leftover playground tests for easier experimenting similar to the test in https://github.com/realm/realm-kotlin/blob/master/packages/cinterop/src/androidTest/java/io/realm/interop/CinteropTest.kt#L32 and could be moved to `cinterop` instead.

Reply:
They have been deleted. We have similar tests in cinterop already and all aspects should be covered by other public API tests. They also caused the Finalizer crash described here https://github.com/realm/realm-kotlin/pull/187#issuecomment-831542798